src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -256,7 +283,7 @@ class CommandRequestImpl : public CommandImpl,
    * interface that returns response.
    * @return resulting code for sending to mobile application.
    */
-  mobile_apis::Result::eType PrepareResultCodeForResponse(
+  virtual mobile_apis::Result::eType PrepareResultCodeForResponse(
```

Comment:
@AByzhynar done in https://github.com/smartdevicelink/sdl_core/pull/1575/commits/596d8b9662b5da0b4c97f5d377d27bdfc0dfe16a